common/src/KokkosKernels_TplsVersion.hpp

```diff
@@ -0,0 +1,58 @@
+//@HEADER
+// ************************************************************************
+//
+//                        Kokkos v. 4.0
+//       Copyright (2022) National Technology & Engineering
+//               Solutions of Sandia, LLC (NTESS).
+//
+// Under the terms of Contract DE-NA0003525 with NTESS,
+// the U.S. Government retains certain rights in this software.
+//
+// Part of Kokkos, under the Apache License v2.0 with LLVM Exceptions.
+// See https://kokkos.org/LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//@HEADER
+
+#ifndef _KOKKOSKERNELS_TPLS_VERSIONS_HPP
+#define _KOKKOSKERNELS_TPLS_VERSIONS_HPP
+
+#include "KokkosKernels_config.h"
+#include <iostream>
+
+#if defined(KOKKOSKERNELS_ENABLE_TPL_CUBLAS)
+#include "cublas_v2.h"
+#endif
+
+#if defined(KOKKOSKERNELS_ENABLE_TPL_CUSPARSE)
+#include "cusparse.h"
+#endif
+
+namespace KokkosKernels {
+
+#if defined(KOKKOSKERNELS_ENABLE_TPL_CUBLAS)
+inline std::string get_cublas_version() {
+  // Print version
+  std::stringstream ss;
+
+  ss << CUBLAS_VER_MAJOR << "." << CUBLAS_VER_MINOR << "." << CUBLAS_VER_PATCH
+     << CUBLAS_VER_BUILD;
```

Comment:
```
(...)/common/src/KokkosKernels_TplsVersion.hpp:39:9: error: use of undeclared identifier 'CUBLAS_VER_BUILD'
     << CUBLAS_VER_BUILD;
```

Reply:
This seems to be present in [CUDA-10.2](https://gitlab.com/nvidia/headers/cuda-individual/cublas/-/blob/cuda-10-2/cublas_api.h#L87). Investigating...

Reply:
Actually we are building against CUDA-10.1.243 which might not be the right thing to do... but in that version of CUDA the variable is not defined, however in CUDA-10.2.089 it is defined indeed